build.sbt

```diff
@@ -94,6 +94,7 @@ lazy val ossFileSystem = (project in file("filesystems/oss"))
 
 lazy val databaseSql = (project in file("database/sql"))
   .withLibrarySettings("cromwell-database-sql", databaseSqlDependencies)
+  .dependsOn(core)
```

Comment:
Pass in the key(s) you need through the database interface. Do not depend on core. Think of this artifact as if you were already in the MySQL/HSQLDB database and had no access to Cromwell business logic.